dateparser/custom_lang_detect/langdetect.py

```diff
@@ -0,0 +1,17 @@
+import langdetect
+from dateparser.conf import apply_settings
+
+langdetect.DetectorFactory.seed = 0
```

Comment:
I think it's bad policy to modify the global state of the library, especially on import. Why do we do this?

Reply:
The library suggests setting this:

```
Language detection algorithm is non-deterministic, which means that if you try to run it on a text which is either too short or too ambiguous, you might get different results everytime you run it.

To enforce consistent results, call following code before the first language detection:

from langdetect import DetectorFactory
DetectorFactory.seed = 0
```

Pl suggest if I should remove this

Reply:
I see, having this deterministic would be great. Is there a way to ensure deterministic results without modifying global state?